test_na_muffn_iris.py

```diff
@@ -0,0 +1,19 @@
+from muffnn import MLPClassifier
```

Comment:
You need to write a real set of tests in the tests directory of the mlp module. Put these tests there in the same format as the ones that are there and then delete this file.